apps/console/src/features/organizations/components/organization-switch/organization-switch-dropdown.tsx

```diff
@@ -239,30 +239,39 @@ const OrganizationSwitchDropdown: FunctionComponent<OrganizationSwitchDropdownIn
                             className="name ellipsis tenant-description"
                             data-componentid={ "organization-dropdown-display-name" }
                         >
-                            { organization?.name ?? (
-                                <Placeholder>
-                                    <Placeholder.Line />
-                                </Placeholder>
-                            ) }
-
-                            { !OrganizationUtils.isRootOrganization(organization) && (
-                                <GenericIcon
-                                    transparent
-                                    inline
-                                    className="manage-tenant-icon"
-                                    data-componentid="associated-component-icon"
-                                    icon={ getSidePanelIcons().serverConfigurations }
-                                    onClick={ (event: SyntheticEvent) => {
-                                        history.push({
-                                            pathname: AppConstants.getPaths()
-                                                .get("ORGANIZATION_UPDATE")
-                                                .replace(":id", organization?.id)
-                                        });
-                                        setIsDropDownOpen(false);
-                                        event.stopPropagation();
-                                    } }
-                                />
-                            ) }
+                            <div>
+                                <div>
+                                    { organization?.name ?? (
+                                        <Placeholder>
+                                            <Placeholder.Line />
+                                        </Placeholder>
+                                    ) }
+                                </div>
+
+                                <Label size="mini">{ organization.id }</Label>
+                            </div>
+
```

Comment:
Let's remove unnecessary new lines.

Reply:
Latest changes removed this section of the component.